librz/core/panels.c

```diff
@@ -4065,7 +4065,7 @@ void __hudstuff(RzCore *core) {
 
 void __esil_init(RzCore *core) {
 	rz_core_analysis_esil_init_mem(core, NULL, UT64_MAX, UT32_MAX);
-	rz_core_cmd(core, "aeip", 0);
+	rz_core_analysis_set_reg(core, "PC", core->offset);
```

Comment:
I think it makes sense to have a `rz_core_analysis_esil_init_regs` function in canalysis.c. We may have to do additional configuration steps on `aeip` than just PC=offset?

Reply:
Done.